### PR TITLE
[SES-268] Custom popper

### PR DESCRIPTION
**Pull request by dvegap95**
# Ticket
https://trello.com/c/OpG0QrVy/268-feature-onchaindatareconciliation-v2

# Description
Create a custom component to serve as common popover component with some of the required features

# What solved
- [x] Should create a basic popover
- [X] Should allow to be opened on hover or on click
- [X] Should be aligned respect to the children

### CODE REVIEW

**Review by alejandrocabriales — APPROVED** [review]
LGTM

File: src/stories/components/CustomTooltip/CustomTooltip.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    (
+      <CustomTooltip
+        content={
+          <Box
+            sx={{
+              border: 1,
+              p: 1,
+              bgcolor: 'background.paper',
+              color: 'text.primary',
+            }}
```
Comment (YasielCabrera) [inline src/stories/components/CustomTooltip/CustomTooltip.stories.tsx:28]: Please, use `styled` over `sx` always if possible

File: src/stories/components/CustomTooltip/CustomTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            color: 'text.primary',
+          },
+        },
+      },
+      onClose: controlledOpen ? () => setControlledOpen(false) : undefined,
+    }),
+    [controlledOpen, enableClickListener]
+  );
+
+  console.log('props', props);
```
Comment (YasielCabrera) [inline src/stories/components/CustomTooltip/CustomTooltip.tsx:46]: is this needed?

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

**Review by YasielCabrera — APPROVED** [review]
LGTM

**Pull request merged**